io.github.ripose.Memento.yml

```diff
@@ -0,0 +1,577 @@
+app-id: io.github.ripose.Memento
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: memento
+rename-desktop-file: memento.desktop
+rename-icon: memento
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
if it supports wayland it should be `fallback-x11`

Reply:
Done